platform/security/handler/security-handler-api/src/main/java/org/codice/ddf/security/handler/api/OidcAuthenticationToken.java

```diff
@@ -13,14 +13,23 @@
  */
 package org.codice.ddf.security.handler.api;
 
+import org.pac4j.core.context.WebContext;
+
 public class OidcAuthenticationToken extends BaseAuthenticationToken {
+  private WebContext webContext;
 
-  public OidcAuthenticationToken(Object credentials, String ip) {
+  public OidcAuthenticationToken(Object credentials, WebContext webContext, String ip) {
```

Comment:
📘 Need the web context in order to resolve credentials later on in the `OidcRealm`.